tools/appveyor.psm1

```diff
@@ -198,8 +198,8 @@ function Invoke-AppVeyorInstall
 
     # Provide credentials globally for remote tests.
     $ss = ConvertTo-SecureString -String $password -AsPlainText -Force
-    $global:AppveyorRemoteCredential = [PSCredential]::new($userName, $ss)
-
+    $appveyorRemoteCredential = [PSCredential]::new("$env:COMPUTERNAME\$userName", $ss)
+	$appveyorRemoteCredential | Export-Clixml -Path "$env:TEMP\AppVeyorRemoteCred.xml" -Force
```

Comment:
If you wanted to use environment variables, the API on AppVeyor is here:  https://www.appveyor.com/docs/build-worker-api/#set-environment-variable